plugins/importexport/native/native.dtd

```diff
@@ -122,7 +122,7 @@
 		public_id CDATA #IMPLIED
 		language CDATA "en">
 
-<!ELEMENT author (firstname, middlename?, lastname, affiliation*, country?, email, url?, competing_interests*, biography*)>
+<!ELEMENT author (firstname*, middlename*, lastname*, affiliation*, country?, email, url?, competing_interests*, biography*)>
```

Comment:
I suspect this should be firstname+, middlename*, lastname+ (at least one first and last name required).